lib/features/manage_account/data/datasource/notification_datasource.dart

```diff
@@ -0,0 +1,7 @@
+abstract class NotificationDataSource {
+  Future<bool> getAppNotificationSetting();
+
+  Future<void> toggleAppNotificationSetting(bool isEnabled);
```

Comment:
```suggestion
  Future<void> toggleAppNotificationSetting({required bool isEnabled});
```
Should use it, for more readable

Reply:
Done